python-threatexchange/threatexchange/signal_type/pdq_ocr.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Wrapper around the Photo PDQ plus OCR signal type.
+Requires the installations of Optical character recognition (OCR) tools
+"""
+
+import math
+import typing as t
+import pathlib
+import warnings
+
+import Levenshtein
+
+from ..descriptor import SimpleDescriptorRollup, ThreatDescriptor
+from . import signal_base
+from ..hashing.pdq_utils import pdq_match
+from .. import common
+
+
+class PdqOcrSignal(signal_base.SimpleSignalType, signal_base.FileMatcher):
+    """
+    PDQ is an open source photo similarity algorithm. See 'pdq.py'
+    This signal type combines pdq hashes with a text string found using
+    optical character recognition (OCR) on the image hashed. This adds an
+    additional 'distance' using levenshtein string comparison.
+
+    Accuracy of matches will depend on quality of OCR implementations and selected thresholds.
+    (ocr_utils uses the open source pytesseract wrapper of tesseract)
+    """
+
+    INDICATOR_TYPE = "HASH_PDQ_OCR"
+    TYPE_TAG = "media_type_photo"
+
+    # This may need to be updated (TODO make more configurable)
```

Comment:
Depending on how we make this more configurable in the future, this could help inform what is needed in the HMA system too. For example, if we find that these settings need to be specific to a collaboration config, then that helps use know that our PDQ matchers in the HMA system also need allow for collab specific settings here. But if we find that it is only user specific, then for the HMA system this could be set as a terraform variable.

Just wanted to point out how the outcome for the cli tool can help us understand the needs for the HMA system too.